src/main/java/com/hivemq/extension/sdk/api/ExtensionMain.java

```diff
@@ -28,53 +28,58 @@
 /**
  * The main starting point for each HiveMQ extension.
  * <p>
- * Each extension must implement at least this interface.
- * The implementation of this interface is used by HiveMQ to load and enable/disable an extension.
+ * Each extension must implement at least this interface. The implementation of this interface is used by HiveMQ to load
+ * and enable/disable an extension.
  *
  * @author Christoph Schäbel
  * @author Georg Held
- * @since 4.0.0
+ * @since 4.0.0, CE 2019.1
  */
 public interface ExtensionMain {
 
     /**
-     * This method is called by HiveMQ if an extension is enabled.<br/>
+     * This method is called by HiveMQ if an extension is enabled.
+     * <p>
      * It can be used to setup the extension and register client interceptors and callbacks with HiveMQ.
      * <p>
      * If this method returns HiveMQ assumes that this extension is finished starting up.
      * <p>
-     * There are multiple options when this method can be called by HiveMQ:<br/>
+     * There are multiple options when this method can be called by HiveMQ:
      * <ul>
-     * <li>When HiveMQ starts up</li>
-     * <li>When the extension is enabled at runtime due to an administrative action</li>
+     *   <li>When HiveMQ starts up
```

Comment:
Why is it removing the closing tags?

Reply:
Javadoc is all about readability (even in the source code)
I read about javadoc coding standards, for example: https://blog.joda.org/2012/11/javadoc-coding-standards.html